src/views/Main/Catalog/Components/PropertyTypeList.js

```diff
@@ -82,13 +86,11 @@ export class PropertyTypeList extends React.Component {
     });
   }
 
-  addProperty = () => (this.props.navBar ? this.createNewPropertyType() : this.addPropertyToSchema());
-
   createNewPropertyType = () => {
     const id = this.props.id;
     const name = document.getElementById('pName'.concat(id)).value;
     const namespace = document.getElementById('pNamespace'.concat(id)).value;
-    const datatype = document.getElementById('pDatatype'.concat(id)).value;
+    const datatype = document.getElementById('pDatatype'.concat(id)).firstChild.firstChild.value;
```

Comment:
`.firstChild.firstChild` is not a good pattern since it relies too heavily on the structure.

what are you trying to do here? are you trying to get a DOM element specifically? is `.firstChild.firstChild` a React component? if it is, you should be interacting with the component, not the DOM.